maps/src/MapBinner.cxx

```diff
@@ -90,6 +92,14 @@ EXPORT_G3MODULE("maps", MapBinner,
 "    stored, though the output maps will *still be weighted*. It should be \n"
 "    set to False if and only if you know what the weights are a priori \n"
 "    (e.g. mock observing, where they are the same as the data maps).\n"
+"map_per_scan: boolean or callable, optional\n"
+"    Defaults to False. If set to True, will make an output map for every \n"
+"    input Scan frame, which can be useful for fine-grained time-domain \n"
+"    maps. Can also be set to a Python callable for complex situations. In \n"
+"    this case, the callable will be called on each Scan frame and passed \n"
+"    the frame as an argument. If it returns True, the map binner will emit \n"
+"    a map frame with the data since the last emitted map frame but *before* \n"
+"    the current Scan; if False, the binner will continue accumulating data.\n"
```

Comment:
In the callable case, doesn't it make more sense to include the current Scan frame instead of emitting only everything that came before?

Reply:
The idea is that this indicates the start of *new* data, which is easier to detect than the end of old data since you don't have to know the future.